backbone/src/main/java/org/researchstack/backbone/model/survey/SurveyItem.java

```diff
@@ -35,18 +35,6 @@
     @SerializedName("items")
     public List<T> items;
 
-    @SerializedName("skipIdentifier")
-    public String skipIdentifier;
-
-    @SerializedName("skipIfPassed")
-    public boolean skipIfPassed;
-
-    // TODO: implement this?
-    public String rulePredicate;  // this is an NSPredicate on iOS, how do we convert?
```

Comment:
I realized there is no such thing as a predicate on Android, so I removed that field.  But the logic behind what it means to have a predicate was moved into expectedAnswer, which you can see in NavigationQuestionStep.